Program.cs

```diff
@@ -0,0 +1,103 @@
+﻿using System;
+using System.IO;
+using System.Collections.Generic;
+
+namespace Zadanie1
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            StreamReader file = new StreamReader("in.txt");
```

Comment:
Trzeba koniecznie robić  using

  **using** StreamReader file = new StreamReader("in.txt");

Inaczej zostaje otwarty handle do strumienia czytającego plik po wykonaniu metody i wtedy dzieją się dziwne rzeczy